Update 2-factor-authentication-on-cloud.md (#4710)
umbraco/UmbracoDocs: umbraco-cloud/set-up/2-factor-authentication-on-cloud.md

meta.Title: 2-factor authentication on Umbraco Cloud
meta.Description: >-
  This article shows you how you can enable 2-factor authentication for when you
  log in to the Umbraco Cloud Portal or the Umbraco Backoffice.
---

# 2-factor authentication

On Umbraco Cloud, you can add 2-factor authentication for your Umbraco Cloud account.

You can use email, phone, or an authenticator app when logging in to the Umbraco Cloud Portal or the Umbraco Backoffice.

{% hint style="info" %}

> Do note that you won't be prompted to authenticate your backoffice login if you've already did it for the portal, as both logins use the same centralized login service.

Reviewer: ```suggestion
You will not be prompted to authenticate your backoffice login if you have already done it for the portal. This is because both logins use the same centralized login service.
```

### Email authentication

> With email authentication, you will get an email with a code that you need to enter when you log in to the Umbraco Cloud portal or the Umbraco Backoffice.

Reviewer: ```suggestion
You will get an email with a code that you need to enter when logging in through the Umbraco Cloud portal or the backoffice.
```
